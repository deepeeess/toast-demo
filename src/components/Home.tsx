import * as React from 'react';

const Home = (props: any) => (
    <div className="home-container" >
    </div >
);

export default Home;