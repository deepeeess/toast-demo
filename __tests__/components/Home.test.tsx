import * as React from 'react';
import Home from '../../src/components/home';
import { shallow } from 'enzyme';
import { } from 'jest';
import toJson from 'enzyme-to-json'

describe('Home', () => {

    it('renders', () => {
        const wrapper = shallow(<Home showSuccess={ jest.fn() } showError={ jest.fn() } />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
    
});