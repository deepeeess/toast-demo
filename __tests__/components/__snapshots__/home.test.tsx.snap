// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders 1`] = `
<div
  className="home-container"
>
  <div
    className="explanation"
  >
    <p>
      Toasts can be dismissed manually by clicking the Ã— icon. Toasts will automatically dismiss after 3,500 milliseconds.
    </p>
    <p>
      The Redux state of the application is logged in the Console.
    </p>
  </div>
  <button
    className="button error"
    onClick={[Function]}
  >
    Show Error Toast
  </button>
  <button
    className="button success"
    onClick={[Function]}
  >
    Show Success Toast
  </button>
  <Connect(Toasts) />
</div>
`;
